web-api/src/business/useCases/trialSessions/getPublicTrialSessionDetailsInteractor.ts

```diff
@@ -9,10 +9,6 @@ export type PublicTrialSessionDetails = Pick<
   | 'trialLocation'
   | 'startDate'
   | 'estimatedEndDate'
-  | 'term'
-  | 'sessionStatus'
-  | 'termYear'
-  | 'sessionType'
```

Comment:
This data is no longer necessary since we do not display the term or session info for public users.